plugins/modules/notification/slack.py

```diff
@@ -100,7 +94,11 @@
   attachments:
     description:
       - Define a list of attachments. This list mirrors the Slack JSON API.
-      - For more information, see also in the (U(https://api.slack.com/docs/attachments)).
+      - For more information, see (U(https://api.slack.com/docs/attachments)).
+  blocks:
+    description:
+      - Define a list of blocks. This list mirrors the Slack JSON API.
+      - For more information, see (U(https://api.slack.com/block-kit)).
```

Comment:
```suggestion
      - For more information, see U(https://api.slack.com/block-kit).
```
URLs should not be in brakets when used as part of a sentence.